src/webgl/loading.js

```diff
@@ -322,6 +436,16 @@ function parseObj(model, lines) {
             face[1] !== face[2]
           ) {
             model.faces.push(face);
+            //same material for all vertices in a particular face
+            const materialDiffuseColor =
```

Comment:
Do we need to add a check here if this exists, so that for objs without materials, we can leave the `vertexColors` array empty?